### DISCUSSION TITLE
장애물 코스 확인/취소 번역 안되어있습니다.

### DISCUSSION THREAD

**Discussion by Rirbe** [wb1016/sb_korpatch#46] [unfinished]
![20220508004630_1](https://user-images.githubusercontent.com/35803731/167261775-f454e0f1-a2ae-4221-afa2-6cf97361b68d.jpg)
Not now 
아직이요.
Ready
준비됬다.

**Comment by wb1016** [comment]
그냥 '아직'이 나아보입니다.
